automated install/basic-install.sh

```diff
@@ -2126,7 +2126,7 @@ checkout_pull_branch() {
 
     git_pull=$(git pull --no-rebase || return 1)
 
-    if [[ "$git_pull" == *"up-to-date"* ]]; then
+    if [[ "$git_pull" == *"up to date"* ]]; then
```

Comment:
```suggestion
    if [[ "$git_pull" == *"up to date"* || "$git_pull" == *"up-to-date"* ]]; then
```

Reply:
An other solution would be to remove the up to date check completly and always indent.

`printf "  %b %s\\n" "${INFO}" "${git_pull}"`